board/drivers/can_common.h

```diff
@@ -11,8 +11,8 @@ typedef struct {
   int8_t forwarding_bus;
   uint32_t can_speed;
   uint32_t can_data_speed;
-  bool canfd_enabled;
-  bool brs_enabled;
+  uint32_t canfd_enabled;
+  uint32_t brs_enabled;
```

Comment:
I think it would be cleaner to just break the cases cppcheck is flagging into if statements and just or them, e.g.
```
if canfd_enabled:
  data |= (1 << 21)
```